.pre-commit-config.yaml

```
@@ -4,6 +4,12 @@ default_language_version:
   python: python3.11
 
 repos:
+  - repo: https://github.com/python-poetry/poetry
+    rev: "1.5.1"
```

quotes `"` aren't necessary

```suggestion
    rev: 1.5.1
```